src/prognostic_equations/surface_temp.jl

```diff
@@ -63,11 +129,16 @@ function surface_temp_tendency!(Yₜ, Y, p, t, slab::PrognosticSurfaceTemperatur
             sfc_turb_w_flux = 0
         end
 
-        # precipitation
+        # 2. Precipitation (rain and snow, defined negative downward, so positive flux 
+        # from surface to atmosphere)
         P_liq = p.precomputed.surface_rain_flux
         P_snow = p.precomputed.surface_snow_flux
 
+        # Total water tendency for surface water:
+        # d(water)/dt = Precip_up + Precip_snow_up + TurbFlux_water_atm_to_sfc
         @. Yₜ.sfc.water -= P_liq + P_snow + sfc_turb_w_flux # d(water)/dt = P - E [kg/m²/s]
-
     end
+
+    return nothing
```

Comment:
I think @dennisYatunin was suggesting not to add extra `return nothing`. Could you clarify?